test/unit/model/test_confidence_interval_applicator.py

```diff
@@ -105,6 +106,23 @@ def test_add_quantiles_to_forecast(self):
         for expected_column in expected_new_columns:
             self.assertTrue(expected_column in pp_forecast.columns)
 
+    def test_add_quantiles_to_forecast_length_mismatch(self):
+        pj = {"quantiles": self.quantiles}
+        pp_forecast = ConfidenceIntervalApplicator(
+            MockModel(), self.stdev_forecast.iloc[:-1, :]  # do not use last value
+        )._add_quantiles_to_forecast_quantile_regression(
```

Comment:
Can you please add the arrange, act assert comments? And maybe A one line docstring about what is tested.

Reply:
Good idea!